Source/Core/DolphinQt2/AboutDialog.cpp

```diff
@@ -39,6 +39,9 @@ AboutDialog::AboutDialog(QWidget* parent) : QDialog(parent)
   // can be downloaded at no cost, but that's not what this message says.)
   text.append(medium + tr("Dolphin is a free and open-source GameCube and Wii emulator.") +
               QStringLiteral("</p>"));
+  text.append(medium +
```

Comment:
Did we do that for any other library we use that has an LGPL license?

Reply:
Yeah, I did thought about that later when some folks explained how it's not an issue if both programs are using the GPL licensing family. But it woud look and feel cleaner if maybe there would be some list, just for user info if not for legal, a button with an additional dialog could be one option which would list all the libraries and could add legal/homepage info/url as an added bonus.

Reply:
This has nothing to do with this PR though and should be discussed separately.